CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+
+Before creating enhancement suggestions, please check [this list](#before-submitting-an-enhancement-suggestion) as you might find out that you don't need to create one. When you are creating an enhancement suggestion, please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill in [the template](ISSUE_TEMPLATE.md), including the steps that you imagine you would take if the feature you're requesting existed.
+
+#### Before Submitting An Enhancement Suggestion
+
+**Perform a [cursory search](https://github.com/minetest-mods/mob-engine/issues?&q=is%3Aissue)** to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/).Create an issue on that repository and provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://daringfireball.net/projects/markdown/syntax#precode).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps or point out the part of Mob Engine which the suggestion is related to. You can use [this tool](http://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
+* **Explain why this enhancement would be useful**
+* **Specify which version of Mob Engine you're using.**
+* **Specify which version of Minetest you're using.**
+
+### Code Contributions
+
+Unsure where to begin contributing to Mob Engine? You can start by looking through these `quicker` and `help-wanted` issues:
+
+* [Quicker issues][quicker] - issues which should only require a few lines of code, and a test or two.
+* [Help wanted issues][help-wanted] - issues which should be a bit more involved than `quicker` issues.
+
+Both issue lists are sorted by total number of comments. While not perfect, number of comments is a reasonable proxy for impact a given change will have.
+
+#### Branching Model
+Mob Engine adopts [Vincent Driessen's branching model](http://nvie.com/posts/a-successful-git-branching-model):
+
+| Branch name | Type | branch off from | merge into | Description |
```

Comment:
So that must be why I do not understand. All the documentation seems well done, but this text is very strange.